src/ds-algorithms/arrays.ts

```diff
@@ -0,0 +1,145 @@
+/*  EXERCISE 1:
+
+    Create a "grades" object that stores a set of student grades in an object.
+    Provide a function for adding a grade and a function for displaying the student's
+    grade average. 
+*/
+
+class Grades {
+    grades!: number[]
+
+    constructor(grades: number[]) {
+        this.grades = grades
+    }
+
+    add(grade: number) {
+        this.grades.push(grade)
+    }
+
+    average(): number {
+        let total: number = 0
+
+        for(let i = 0; i < this.grades.length; i++) {
+            total += this.grades[i]
+        }
+
+        return total / this.grades.length
+    }
+}
```

Comment:
#Advice:
- It will be interesting to use encapsulation here

Reply:
Here you have an example: https://1.bp.blogspot.com/-dDN_hHIh6H8/XRSoOYMJtpI/AAAAAAAAGJI/5y9-J8eEe4AIeGrRIEJBTlZIpUrKmbMeQCLcBGAs/s1600/generate-getter-setter-typescript.gif

Reply:
Every time you finish your comment fixes you can hit resolve button

Reply:
Ok, noted 👍